_docs/setup/kubernetes/sidecar-injection.md

```diff
@@ -100,7 +100,7 @@ kubectl apply -f <(~istioctl kube-inject -f samples/sleep/sleep.yaml)
 Verify that the sidecar has been injected into the deployment.
 
 ```
-$ kubectl get deployment sleep -o wide
+kubectl get deployment sleep -o wide
```

Comment:
Please move the command output to a separate block here and elsewhere.

Reply:
Fixed in a subsequent PR, not sure how to update one created from "Edit this file"

Reply:
click edit when in your branch